lib/test/t_LinearModelAlgorithm_std.cxx

```diff
@@ -70,6 +70,23 @@ int main(int, char *[])
     LinearModelAlgorithm test(X, Y);
     LinearModelResult result(test.getResult());
     fullprint << "trend coefficients = " << result.getCoefficients() << std::endl;
+
+    // Test various attributes
+    Point cook(result.getCookDistances());
+    assert_almost_equal(cook[0], 0.0233296, 1e-5, 0.0);
+    assert_almost_equal(cook[1], 0.0360369, 1e-5, 0.0);
+    assert_almost_equal(cook[2], 0.00178903, 1e-5, 0.0);
+    assert_almost_equal(cook[3], 0.0502183, 1e-5, 0.0);
+    assert_almost_equal(cook[4], 0.0966701, 1e-5, 0.0);
+    assert_almost_equal(cook[4], 0.00562596, 1e-5, 0.0);
+
+    Point leverages(result.getLeverages());
+    assert_almost_equal(leverages[0], 0.0864939, 1e-6, 0.0);
+    assert_almost_equal(leverages[1], 0.0797831, 1e-6, 0.0);
+    assert_almost_equal(leverages[2], 0.0735447, 1e-6, 0.0);
+    assert_almost_equal(leverages[3], 0.0677578, 1e-6, 0.0);
+    assert_almost_equal(leverages[4], 0.0624023, 1e-6, 0.0);
+    assert_almost_equal(leverages[4], 0.0574582, 1e-6, 0.0);
```

Comment:
I'd rather issue one single call to assert_almost_equal on a Point
you can use "cont Point cook_reference = {0.086, 0.079, ...}"

Reply:
Thanks for the tip! Done.